magda-registry-api/README.md

```diff
@@ -51,7 +51,7 @@ users in different organization units. See the comment in the sql file for detai
 ### Start opa server
 
 ```
-cd deploy/helm/magda-core/charts/opa
+cd deploy/helm/magda/charts/magda-core/charts/opa
```

Comment:
`magda-core` chart files are not in `magda` 's chart directory so original path should be correct?

Reply:
Oops yeah... I moved them and moved them back, looks like I missed a spot. Good catch :)